modules/osdk-multi-arch-building-images.adoc

```diff
@@ -0,0 +1,125 @@
++
+The platforms that your Operator and operands mutually support determine the platform compatibility of your Operator project.
++
+.Example output
+[source,json]
+----
+{
+    "manifests": [
+	{
+	    "digest": "sha256:c0669ef34cdc14332c0f1ab0c2c01acb91d96014b172f1a76f3a39e63d1f0bda",
+	    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+	    "platform": {
+		"architecture": "amd64",
+		"os": "linux"
+	    },
+	    "size": 528
+	},
+...
+	{
+	    "digest": "sha256:30e6d35703c578ee703230b9dc87ada2ba958c1928615ac8a674fcbbcbb0f281",
+	    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+	    "platform": {
+		"architecture": "arm64",
+		"os": "linux",
+		"variant": "v8"
+	    },
+	    "size": 528
+	},
+...
+----
+
+** If the previous command does not output platform information, then the specified base image might be a single image instead of an image manifest. You can find which architectures an image supports by running the following command:
++
+[source,terminal]
+----
+$ docker inspect <image>
+----
+
+. For Go-based Operator projects, the Operator SDK explicitly references the `amd64` architecture in your project's Dockerfile. Make the following change
+to your Dockerfile to set an environment variable to the value specified by the platform flag:
++
+.Example Dockerfile
+[source,docker]
+----
+FROM golang:1.19 as builder
+ARG TARGETOS
+ARG TARGETARCH
+...
+RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go <1>
+----
+<1> Change the `GOARCH` field from `amd64` to `$TARGETARCH`.
+
+. Your Operator project's makefile defines the `PLATFORM` environment variable. When you run the `make docker buildx` command to generate a manifest list, the Operator SDK creates an image manifest for each of the platforms specified by the variable.
++
+.Example makefile
+[source,make]
+----
+# ...
+PLATFORMS ?= linux/arm64,linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
+# ...
+----
++
+You can edit the `PLATFORM` environment variable to specify the image manifests included in your manifest list. The following example defines the supported platforms as `linux/arm64` and `linux/amd64`:
++
+.Example makefile
+[source,make]
+----
+# ...
+PLATFORMS ?= linux/arm64,linux/amd64
+.PHONY: docker-buildx
+# ...
+----
++
+[TIP]
+====
+You can change the `PLATFORMS` environment variable without editing your Operator project's makefile. Run the following command to temporarily override the `PLATFORMS` environment variable:
+
+[source,terminal]
+----
+$ make docker-buildx PLATFORMS=linux/amd64,linux/arm64 \
+  IMG=<image_registry>/<organization_name>/<repository_name>:<version_or_sha>
+----
+
+If you edit the `PLATFORMS` environment variable from the CLI, skip the next step in this procedure.
+====
+
+. Run the following command from your Operator project directory to build a manager image to cross-compile support for multiple platforms and push the manifest list your registry:
```

Comment:
Good point. I will break it into smaller sentences

Reply:
The command does a kitchen sink of functions, and the fact that it will build and push the image isn't obvious from the command syntax. I will finesse this a bit.

Reply:
Please let me know if the updated wording is clearer.

Reply:
In terms of sentence structure, it's a much much easier read after your update.